impeller/entity/contents/text_contents.cc

```diff
@@ -170,22 +170,24 @@ bool TextContents::Render(const ContentContext& renderer,
       vertex_count * sizeof(VS::PerVertexData), alignof(VS::PerVertexData),
       [&](uint8_t* contents) {
         VS::PerVertexData vtx;
-        size_t vertex_offset = 0;
-        for (const auto& run : frame_.GetRuns()) {
+        VS::PerVertexData* vtx_contents =
+            reinterpret_cast<VS::PerVertexData*>(contents);
+        for (const TextRun& run : frame_.GetRuns()) {
           const Font& font = run.GetFont();
-          auto rounded_scale = TextFrame::RoundScaledFontSize(
+          Scalar rounded_scale = TextFrame::RoundScaledFontSize(
```

Comment:
Is this actually doing anything or just a style preference?

Reply:
It's the C++ style guide: https://google.github.io/styleguide/cppguide.html#Type_deduction

> Use type deduction only if it makes the code clearer to readers who aren't familiar with the project, or if it makes the code safer. Do not use it merely to avoid the inconvenience of writing an explicit type.